src/main/java/testUI/TestUIDriver.java

```diff
@@ -225,4 +222,43 @@ public static DesiredCapabilities getDesiredCapabilities() {
     public static UIElement getTestUIDriver() {
         return TestUI.E("");
     }
+
+    public static void switchApplicationContext(String context) {
+        AppiumDriver driver = getDriver();
+
+        if (driver instanceof AndroidDriver) {
+            AndroidDriver androidDriver = (AndroidDriver) driver;
+            Set<String> contextNames = androidDriver.getContextHandles();
+
+            boolean contextFound = false;
+            for (String contextName : contextNames) {
+                if (contextName.contains(context)) {
+                    androidDriver.context(contextName);
+                    System.out.println("Switched to context: " + contextName);
+                    contextFound = true;
+                    break;
+                }
+            }
+            if (!contextFound) {
+                System.out.println("Provided context is not available");
```

Comment:
for the System.out.println, it would be better to use the logger method:

`putLogInfo("Provided context is not available");`

You will also have to import it:

`import static testUI.Utils.Logger.putLogInfo;`